kwok.yaml

```diff
@@ -0,0 +1,11 @@
+kind: KwokctlConfiguration
+apiVersion: config.kwok.x-k8s.io/v1alpha1
+options:
+  etcdPort: 2379
+  etcdPrefix: /kube-scheduler-simulator
+  disableKubeScheduler: true
+componentsPatches:
+- name: kube-apiserver
+  extraArgs:
+  - key: cors-allowed-origins
+    value: ^*$
```

Comment:
Seemingly it's not working. @wzshiming Any idea?

You should be able to reproduce it by `make docker_up_local` and visit localhost:3000 (web ui).

<img width="606" alt="スクリーンショット 2024-03-22 23 47 24" src="https://github.com/kubernetes-sigs/kube-scheduler-simulator/assets/44139130/e2344574-b4ab-4234-bcfc-05a8206b8f64">

Reply:
I did some debugging

in v0.1.1
```
curl http://localhost:3131/api/v1/namespaces//pods

{
    "kind": "PodList",
    "apiVersion": "v1",
    "metadata": {
        "resourceVersion": "215"
    },
    "items": []
}
```

in this pr
```
curl http://localhost:3131/api/v1/namespaces//pods

# 301 to /api/v1/namespaces/pods

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "namespaces \"pods\" not found",
  "reason": "NotFound",
  "details": {
    "name": "pods",
    "kind": "namespaces"
  },
  "code": 404
}
```

Getting all pods might look like this

```
curl http://localhost:3131/api/v1/pods
```

Reply:
Interesting but that one may be a separated another issue from CORS one that I shew? 
I think CORS issue is specific to web browser.

Reply:
No, get fails, but it can create it. It's not a CORS problem.

I tried booting all the way up to 0.1.1 first, bypassing the initialization, and then later uninstalling 0.1.1, and then booting this PR version, and it worked.

Reply:
Or let's try putting node or ns first, and we'll know if it's a CORS problem.

Reply:
<img width="1315" alt="image" src="https://github.com/kubernetes-sigs/kube-scheduler-simulator/assets/6565744/4572fe0c-a703-4330-ba0a-d554da42cbad">

Strangely, after I rebuild the frontend, the path works, but the simulator's CORS is broken.

Reply:
Oh, I typed the address. It should be http://localhost:3000/. It's working now. just rebuild frontend.

Reply:
Well, so my CORS issue isn't reproducible. I have to recheck things on my side again then.

Reply:
<img width="1005" alt="スクリーンショット 2024-03-26 9 00 22" src="https://github.com/kubernetes-sigs/kube-scheduler-simulator/assets/55221074/c8df7abe-f367-4b0c-a1db-99c4ab5d49fc">

Thank you for investigating this matter. I tried to reproduce on my environment, but I couldn't.